covid_plot.py

```diff
@@ -272,7 +272,12 @@ def plot_area(df: pd.DataFrame,
             df_plot.plot(ax=a1, y=perccols, kind='area', legend=False)
             a1.set_ylabel('Percent')
             a1.xaxis.label.set_visible(False)
+            a1.secondary_yaxis('right', functions=( lambda x: x, lambda x: x ))
```

Comment:
Sorry to be pedantic about formatting but it throws a red a squiggly in our IDEs: please could you remove the spaces between the lambdas and the parentheses  on this and the next line? Thanks!